.bazelrc

```diff
@@ -25,19 +25,19 @@ build --action_env=CC
 build --action_env=CXX
 build --action_env=PATH
 
+# Common flags for sanitizers
+build:sanitizer --define tcmalloc=disabled
+build:sanitizer --linkopt -ldl
+build:sanitizer --build_tag_filters=-no_san
+build:sanitizer --test_tag_filters=-no_san
+
 # Basic ASAN/UBSAN that works for gcc
-build:asan --action_env=BAZEL_LINKLIBS=
-build:asan --action_env=BAZEL_LINKOPTS=-lstdc++:-lm
+build:asan --config=sanitizer
 build:asan --define ENVOY_CONFIG_ASAN=1
 build:asan --copt -fsanitize=address,undefined
 build:asan --linkopt -fsanitize=address,undefined
 build:asan --copt -fno-sanitize=vptr
 build:asan --linkopt -fno-sanitize=vptr
-build:asan --linkopt -ldl
-build:asan --define tcmalloc=disabled
-build:asan --build_tag_filters=-no_asan
-build:asan --test_tag_filters=-no_asan
-build:asan --define signal_trace=disabled
```

Comment:
Did you mean to lose this? I don't think this is replaced elsewhere?

Reply:
I don't know why we did need this originally because it doesn't fail, what is the original reason for this? For ASAN symbolizer? TSAN worked well without this.

Reply:
I have no idea, I just noticed that it was removed here and I wanted to make sure you meant to do it. Seems fine to remove.

Reply:
let me do a quick try on ASAN failure (ASSERT(false)) to see if signal_trace does something wrong.

Reply:
ah yes it eats the stacktrace in case of SIGABRT though this stinks a bit (`nested bug` means we might have asan fault in the signal tracer but not sure), bring it back in this PR.

```
[2019-08-23 21:27:48.904][20199][critical][assert] [test/server/server_test.cc:40] assert failure: false.
[2019-08-23 21:27:48.904][20199][critical][backtrace] [bazel-out/k8-fastbuild/bin/source/server/_virtual_includes/backtrace_lib/server/backtrace.h:81] Caught Aborted, suspect faulting address 0x3e900004ee7
[2019-08-23 21:27:48.904][20199][critical][backtrace] [bazel-out/k8-fastbuild/bin/source/server/_virtual_includes/backtrace_lib/server/backtrace.h:69] Backtrace (use tools/stack_decode.py to get line numbers):
=================================================================
AddressSanitizer: nested bug in the same thread, aborting.
================================================================================
```